pandasai/__init__.py

```diff
@@ -196,13 +199,17 @@ def run_code(
             # Execute the code
             count = 0
             code_to_run = self.remove_unsafe_imports(code)
+            _success_in_running_code = False
             while count < self._max_retries:
                 try:
+                    if any([x in code_to_run for x in PLOT_KEY_WORDS]) and code_to_run.count(PLT_CODE) < 1:
```

Comment:
What if it is already showing the plot? We show it again? remember it could show with `block= False` or not being passed, so let's make sure all the cases are considered